crates/biome_html_formatter/src/html/lists/attribute_list.rs

```diff
@@ -6,7 +6,8 @@ pub(crate) struct FormatHtmlAttributeList;
 impl FormatRule<HtmlAttributeList> for FormatHtmlAttributeList {
     type Context = HtmlFormatContext;
     fn fmt(&self, node: &HtmlAttributeList, f: &mut HtmlFormatter) -> FormatResult<()> {
-        let line_break = if f.options().attribute_position() == AttributePosition::Multiline {
+        let attribute_len = node.iter().len();
+        let line_break = if attribute_len > 1 || f.options().attribute_position() == AttributePosition::Multiline {
```

Comment:
This condition should only apply when in multiline mode.
```suggestion
        let line_break = if f.options().attribute_position() == AttributePosition::Multiline && attribute_len > 1 {
```